.gitlab/ascent.yml

```diff
@@ -11,6 +11,9 @@ ascent_pr_regression_test:
   stage: regression
   rules:
     - if: '$CI_PIPELINE_SOURCE == "external_pull_request_event" && $CI_EXTERNAL_PULL_REQUEST_TARGET_BRANCH_NAME == $CI_DEFAULT_BRANCH'
+      changes:
+        - $CI_PROJECT_DIR/buildtest/buildtest/**/*
+        - $CI_PROJECT_DIR/buildtest/tests/**/*
```

Comment:
just remove the $CI_PROJECT_DIR since the `changes` field assume relative file path is starting from root of repo i.e `CI_PROJECT_DIR`

Reply:
@shahzebsiddiqui Done. You can merge this in.